akka-projection-grpc/src/main/scala/akka/projection/grpc/internal/EventProducerServiceImpl.scala

```diff
@@ -384,4 +387,23 @@ import akka.projection.grpc.producer.scaladsl.EventProducer.Transformation
       }
     }
   }
+
+  override def latestEventTimestamp(
+      req: LatestEventTimestampRequest,
+      metadata: Metadata): Future[LatestEventTimestampResponse] = {
+    intercept(req.streamId, metadata).flatMap { _ =>
+      val producerSource = streamIdToSourceMap(req.streamId)
+      eventsBySlicesPerStreamId(req.streamId) match {
+        case q: LatestEventTimestampQuery =>
```

Comment:
This is why we need the query interface in Akka core. gRPC projection doesn't have a dependency to r2dbc